apps/estuary/assets/css/app.scss

```diff
@@ -0,0 +1,134 @@
+@import "./variables.scss";
```

Comment:
There is a lot of unused styling in here. At some point, we should probably share this styling somehow between this and ANDI. Until then, I'm fine having these extras here for consistency and future use.